projects/1.20/assets/relics-vivid-light-addon/relics_vivid_light/lang/zh_cn.json

```
@@ -0,0 +1,67 @@
+  "tooltip.relics.static_rifle.ability.bzzshot.stat.static_duration.value" : "%1$s秒。",
+  "tooltip.relics.static_rifle.ability.forcefield" : "力场",
+  "tooltip.relics.static_rifle.ability.forcefield.description" : "静电步枪会压缩能量，产生保护性力场。力场之内的敌人均会受到静电效果影响。冲击性的防御策略，自静电步枪的震感力场而来！",
+  "tooltip.relics.static_rifle.ability.forcefield.stat.radius.title" : "半径：",
+  "tooltip.relics.static_rifle.ability.forcefield.stat.radius.value" : "%1$s格方块。",
+  "tooltip.relics.static_rifle.ability.forcefield.stat.duration.title" : "持续时间：",
+  "tooltip.relics.static_rifle.ability.forcefield.stat.duration.value" : "%1$s秒。",
+  "tooltip.relics.static_rifle.ability.forcefield.stat.static_probability.title" : "静电效果概率：",
+  "tooltip.relics.static_rifle.ability.forcefield.stat.static_probability.value" : "%1$s%%每刻。",
+  "item.relics_vivid_light.morphcook" : "变形厨师",
+  "tooltip.relics.morphcook.leveling" : "变形厨师，小小的厨艺大师，能将寻常材料化作可口的菜肴。要问独特的能力？那就是把各式血肉变成营养丰富的大餐。出于寻常而超乎寻常，厨艺之旅自此起始！",
+  "tooltip.relics.morphcook.ability.passive_smoking" : "被动烹饪",
+  "tooltip.relics.morphcook.ability.passive_smoking.description" : "变形厨师的法力起了作用，尚未加工的物件直接变形为了烹煮精制的珍馐。效率与烟熏炉类似，变形厨师越多就越高效。有了这位变形好搭档，普通的厨房就成了厨业工厂！",
+  "tooltip.relics.morphcook.ability.fleshfeeding" : "餐食血肉",
+  "tooltip.relics.morphcook.ability.fleshfeeding.description" : "\n变形厨师从周围的生物搜刮来少量的肉，以此制作一道沁人心脾的佳肴。便捷性和营养性配合得恰到好处，不愧为厨艺奇迹！",
+  "item.relics_vivid_light.whirlwind_spear" : "漩流长矛",
+  "tooltip.relics.whirlwind_spear.leveling" : "漩流长矛，自海底深渊取回的神秘武器，是潜入深溟的不二之选。深海的精华涌过矛身，似在邀请你掌控构筑其形体的洋流。陷于战斗的涡流之时，这柄长矛即是海洋之力伴随你身，如同海神本尊倾泻着他的威能。",
+  "tooltip.relics.whirlwind_spear.ability.whirl_swirl" : "涡旋海流",
+  "tooltip.relics.whirlwind_spear.ability.whirl_swirl.description" : "漩流长矛召唤出如在暴怒的海洋风暴，将周围的敌人拖入涡流的怀抱。落入乱流的敌人会逐渐没入水底，不管如何拖延，溺水终究不可避免。呼风唤雨，海洋本身也成了可怖的武器，无往而不利。",
+  "tooltip.relics.whirlwind_spear.ability.whirl_swirl.stat.drowning_speed.title" : "溺水速度：",
+  "tooltip.relics.whirlwind_spear.ability.whirl_swirl.stat.drowning_speed.value" : "%1$s点速度。",
+  "tooltip.relics.whirlwind_spear.ability.whirl_swirl.stat.radius.title" : "半径：",
+  "tooltip.relics.whirlwind_spear.ability.whirl_swirl.stat.radius.value" : "%1$s格方块。",
+  "tooltip.relics.whirlwind_spear.ability.blob_a_bubble" : "泡泡护罩",
+  "tooltip.relics.whirlwind_spear.ability.blob_a_bubble.description" : "漩流长矛的持有者被无法穿透的泡泡包裹，近乎无懈可击。保护带来的是坚定不移，不论何种攻击都无法撼动身处其中的玩家。信赖漩流长矛的保护能力的水能护盾吧！",
+  "tooltip.relics.whirlwind_spear.ability.blob_a_bubble.stat.duration.title" : "持续时间：",
+  "tooltip.relics.whirlwind_spear.ability.blob_a_bubble.stat.duration.value" : "%1$s秒。",
+  "tooltip.relics.whirlwind_spear.ability.blob_a_bubble.stat.cooldown.title" : "冷却：",
+  "tooltip.relics.whirlwind_spear.ability.blob_a_bubble.stat.cooldown.value" : "%1$s秒。",
+  "effect.relics_vivid_light.static" : "静电",
+  "item.relics_vivid_light.steam_boiler" : "蒸汽锅炉",
+  "tooltip.relics.steam_boiler.leveling" : "\n蒸汽锅炉，由坚韧的铜锻成，是紧凑而精密的机械奇迹。体积不大，但能力极强；紧随蒸汽的能量而来的是极高的效率和可靠性。坚固的设计和稳定的运行表现，便是这件金属造物为你的冒险准备的牢靠基础。",

```

powerful可用于形容机械工具的性能
```suggestion
  "tooltip.relics.steam_boiler.leveling" : "\n蒸汽锅炉，由坚韧的铜锻成，是紧凑而精密的机械奇迹。体积不大，但性能极强；紧随蒸汽的能量而来的是极高的效率和可靠性。坚固的设计和稳定的运行表现，便是这件金属造物为你的冒险准备的牢靠基础。",
```